chat/receipt_handlers.py

```diff
@@ -63,6 +112,76 @@ def __fetch_receipts(messages, user_id):
     return found_receipts
 
 
+def __update_user_conversations(unread_counts, last_read_messages, conn):
+    """
+    Update user conversation table based on un-read counts and
+    last read messages
+    """
+    user_id = current_user_id()
+    schema_name = AsIs(_get_schema_name())
+    for key in unread_counts:
```

Comment:
`unread_counts` -> `unread_counter`?

I think give a hint annotation on telling `last_read_messages` is a dict will help other understands.